src/i18n/en/project-home.json

```diff
@@ -7,7 +7,7 @@
   "Clear": "Clear",
```

Comment:
I added DE labels.

Just an additional note: the DE labels are not automatically filled in, if only the EN labels exist. I.e. if there are no DE labels, UI elements will be blank in the DE interface. (We should always fill the DE labels, even if we temporarily only fill them with the English versions.)